core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -115,6 +115,17 @@ public static String getStringOrNull(String property, JsonNode node) {
     return pNode.asText();
   }
 
+  public static JsonNode getChildNodeOrNull(String property, JsonNode node) {
+    if (!node.has(property)) {
+      return null;
+    }
+    JsonNode pNode = node.get(property);
```

Comment:
Style: control flow blocks should be separated from the next statement by an empty newline.